test/test.go

```diff
@@ -0,0 +1,4 @@
+package test
+
+// We have two arrays of key value pair.
+// We want to replace value in one of the array with matching key in another.
```

Comment:
do we need this file ?

Reply:
I think this came by mistake. Deleting it.